packages/language-server/src/server.ts

```diff
@@ -174,3 +208,7 @@ documents.onDidClose((e) => {
 documents.listen(connection);
 
 connection.listen();
+
+function isXMLView(uri: string): boolean {
+  return /(view|fragment)\.xml/i.test(uri);
```

Comment:
remove case insensitive flag and use end of input anchor
to ensure .xml is end of file name.

Reply:
the goal is to align this regexp with the documentSelector pattern in the client.